src/useIsShake.ts

```diff
@@ -51,14 +54,41 @@ export function useIsShake() {
     setSubscription(null);
   };
 
+  const checkAccelerometerAvail = async () => {
+    const availability = await isAvailableAsync();
+    setIsAccelerometerAvail(() => {
+      setIsAccelerometerAvailPending(false);
+      return availability;
+    });
+  };
+
+  const getAccelerometerPermiss = async () => {
```

Comment:
The added `ion` to `Permiss` is worth it. 
```suggestion
  const getAccelerometerPermission = async () => {
```